src/main/app/claims/claimStoreClient.ts

```diff
@@ -7,7 +7,7 @@ import { ClaimModelConverter } from 'claims/claimModelConverter'
 import { ResponseModelConverter } from 'claims/responseModelConverter'
 
 export const claimApiBaseUrl = `${config.get<string>('claim-store.url')}`
-const claimStoreApiUrl = `${claimApiBaseUrl}/claims`
+export const claimStoreApiUrl = `${claimApiBaseUrl}/claims`
```

Comment:
I'd add type as it's now exported. Actually types could be added to all consts here.

NOT REQUEST. To consider :)